app/routes/draft.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const draftController = require('../controllers/draft');
+
+const express = require('express');
+let router = express.Router();
+
+// POST new draft
+router.post('/', draftController.getDraft);
```

Comment:
This route needs the authentication middleware attached to it. Check out the `user` route for an example